### PR TITLE
Add ability to set custom picture for entities

### PR DESCRIPTION
**Pull request by thybag**
Hi @nathan-gs,
Hope these look okay, two  minor quality of life additions. 

* Add `picture` property to entities. Setting this with a URL will override the default photo for markers.
* ~Add `map_height` (int) to card. This sets map min-height and can be used to override/force a preferred height for the map itself.~

### CODE REVIEW

File: map-card.js [hunk 1]
```diff
@@ -149,7 +151,8 @@ class MapConfig {
   tileLayers;
   /** @type {TileLayerConfig} */
   tileLayer;
-
+  /** @type {Int} */
+  map_height;
```
Comment (nathan-gs) [inline map-card.js:155]: let's use camelCase here

Comment (thybag) [reply map-card.js]: Ah, good catch. Have updated 👍

**Comment by nathan-gs** [comment]
Do we really need a `map_height` vs `card_size` option?

**Comment by thybag** [comment]
Hi @nathan-gs, I'd missed the `card_size` when i first added. I agree its pretty much a duplication so have removed from the PR 👍

**Pull request merged**